auth-svc/service/db-service/UserRepository.js

```diff
@@ -0,0 +1,32 @@
+const { PrismaClient } = require('@prisma/client');
+
+const prisma = new PrismaClient();
+
+class UserRepository {
+  async createUser(registrationPayload) {
+    const user = await prisma.users.create({
+      data: {
+        email: registrationPayload.email,
```

Comment:
are there any fields on the `registrationPayload` that is not listed here? if not, since the keys of the registration payload match your data input you could just do `primsa.users.create({ data: registrationPayload })` right?

Reply:
are you guys going to use typescript in this project btw? i've noticed everything here is strictly javascript but i don't see many companies hiring purely node.js without typescript.

Reply:
Yeah, that's true, I'll fix it.
About TS I built it before we decided to use TS in a project, and then I thought that it'd be easier to first refactor the code and then migrate it to TS.
Or what do you think I should do?

Reply:
the word migrations can strike fear in many engineers eyes. sometimes its necessary but when you can you usually try to avoid it. i think it'd be better to have typescript earlier than later. that way you can also have more time coding with it.

Reply:
I can only imagine how painful it could be on a big project.
So I got it. It'll be my next goal.

Reply:
yeah do it in a separate ticket/branch after all your stuff gets merged.